doc/rtd/topics/tutorials/qemu.rst

```diff
@@ -0,0 +1,140 @@
+.. _tutorial_qemu:
+
+Qemu
```

Comment:
We may need to make the title a bit more descriptive. Where it appears in the LHS menu as just "Qemu", a new user may not know what it is or whether it's relevant to their interests.

Reply:
+1